jpa/hibernate7/src/main/java/org/wildfly/persistence/jipijapa/hibernate7/VirtualFileSystemArchiveDescriptor.java

```diff
@@ -82,4 +82,34 @@ public InputStreamAccess getStreamAccess() {
             archiveContext.obtainArchiveEntryHandler( entry ).handleEntry( entry, archiveContext );
         }
     }
+
+    //  Example call from org.hibernate.boot.archive.scan.internal.DisabledScanner
+    //  final ArchiveEntry entry = archiveDescriptor.findEntry( "META-INF/orm.xml" );
+    public ArchiveEntry findEntry(String path) {
```

Comment:
Minor nit, but IMO this should have the `@Override` annotation.

Reply:
+1 I'll add that after current testing completes.